chains/evm/calls/transactor/monitored/monitored.go

```diff
@@ -81,36 +93,44 @@ func (t *MonitoredTransactor) Transact(to *common.Address, data []byte, opts tra
 			return &common.Hash{}, err
 		}
 	}
+	if len(gp) > 1 {
```

Comment:
It does not make sense to me to add logic to log something when if you just log the whole gas price it is clear from the values what it is.

Reply:
Yeah that makes sense